### ISSUE TITLE
[Bug]: You cannot eat with a wooden hand.

### ISSUE THREAD

**Issue by JohnnyJohnJohnson** [Darkrp-community/OpenKeep#136]
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

RogueTown

### Describe the issue

When I try to eat with a wooden hand, it still says "You cannot move the fingers." despite being able to do everything else with a wooden hand so far.

### How to reproduce

Lose an arm, get a wooden arm, try to eat something held in that arm. I lost both my arms and got two wooden ones, I don't know if it works if you have one wooden arm and one real one.

### Round ID

Dragon89-20.09.18

**Comment by Stutternov** [comment]
Weirdly this is intended in the fingers variable, which the wooden hand lacks, though I may change that if it’s wanted by devs. It does feel a bit limiting. 

Idea basically is the wooden hand can be used to hold stuff but ‘isn’t made to be used’ (beyond helping you two hand or just temp hold stuff)

**Comment by JohnnyJohnJohnson** [comment]
> Weirdly this is intended in the fingers variable, which the wooden hand sets purposefully to false, though I may change that if it’s wanted by devs. It does feel a bit limiting.
> 
> Idea basically is the wooden hand can be used to hold stuff but ‘isn’t made to be used’ (beyond helping you two hand or just temp hold stuff)

Yeah but eating and drinking don't feel like you'd need your fingers necessarily. If you can raise it to your mouth you should be able to put it in your mouth, same with liquid, just tilt your body back. It's easier than trying to craft stuff. Also I can swing a weapon with a wooden arm, but I can't chop a tree.
